ui/src/publication/log/publication-log.tsx

```diff
@@ -24,8 +24,10 @@ import { useAppNavigate } from 'common/navigate';
 import { defaultPublicationSearch } from 'publication/publication-utils';
 import { DOWNLOAD_PUBLICATION } from 'user/user-model';
 import { Spinner } from 'vayla-design-lib/spinner/spinner';
+import { debounceAsync } from 'utils/async-utils';
 
 let fetchId = 0;
+const debouncedGetPublicationsAsTableItems = debounceAsync(getPublicationsAsTableItems, 500);
```

Comment:
Debounce julkaisulokin hakemiselle. Jossain muualla oli käytetty 250 millisekuntia, mutta omalla testailulla 500 tuntui järkevämmältä raja-arvolta. Koska kyseessä on numerokenttä niin sinne kirjoittaminen on ainakin allekirjoittaneella hieman normitekstiä hitaampaa, ja sitä myötä pienempi raja-arvo (250ms) tuntui johtavan turhiin kutsuihin. Vastaavasti isompi raja-arvo (1000ms) tekee käytöstä todella hidasta

Reply:
Olikos debounce käytössä myös muulle kuin kirjoitusvalinnalle? Sinällään ajattelisin, että datepickeristä valitsemalla käyttäjä osui kyllä jo oikeaan päivämäärään jolloin 500ms lisäviive olisi jo aika pitkä lisäviive 🤔. Tosin eivät ne kutsut ihan heti palaudu kyllä bäkkäriltäkään.

Reply:
Debounce on käytännössä molemmille, sillä DatePicker exposaa toistaiseksi ainoastaan päivämäärän mutta ei sitä mitä kautta se asetettiin. Nyt kun tätä mielessäni pöyhäisin, tuon noiden tilanteiden eriyttäminen ei välttämättä olisi lopulta kauhean hankalaa, mutta se vaatii jonkinlaista säätöä datepickerin API:in. Pistetään korvan taakse review-korjausehdotukseksi 🤔